### ISSUE TITLE
crash with digested bin file

### ISSUE THREAD

**Issue by ezorita** [mirnylab/cooler#32]
Are there any known issues with `cooler cload` when using non-fixed bin sizes?

I am trying to generate a cooler file with a digested bin file:

`cooler digest genomes/hg19_chr_only_length.txt genomes/hg19_chr_only.fasta MboI > bins/hg19_mboi_1f.bed`

and then 

`cooler cload tabix bins/hg19_mboi_1f.bed hic_data.pairs.gz hic_data_1f.cool`

after a few minutes..

```
chroms
bins
pixels
Traceback (most recent call last):
  File "/home/ezorita/miniconda2/bin/cooler", line 11, in <module>
    load_entry_point('cooler', 'console_scripts', 'cooler')()
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/mnt/Data/BHIVE/HiC/pipeline/cooler/cooler/cli/cload.py", line 155, in tabix
    create(h5, chroms, lengths, bins, iterator, metadata, assembly)   
  File "/mnt/Data/BHIVE/HiC/pipeline/cooler/cooler/io/__init__.py", line 67, in create
    bin1_offset, nnz = write_pixels(grp, n_bins, reader, h5opts)
  File "/mnt/Data/BHIVE/HiC/pipeline/cooler/cooler/io/_writer.py", line 157, in write_pixels
    for chunk in reader:
  File "/mnt/Data/BHIVE/HiC/pipeline/cooler/cooler/io/_reader.py", line 250, in __iter__
    for df in self.aggregate(map=pool.imap):
  File "/home/ezorita/miniconda2/lib/python2.7/site-packages/multiprocess/pool.py", line 668, in next
    raise value
TypeError: range() integer step argument expected, got NoneType.
```

I generated several cooler files with the same dataset `hic_data.pairs.gz` using fixed bin sizes without any problem.

Cheers

**Comment by nvictus** [comment]
You're right. Non-fixed binning on tabix/pairix broke recently. Should be resolved in 0.6.3. If you still run into issues, I'll re-open.

**Comment by ezorita** [comment]
Works fine, thanks.
